utils/fasm/src/fasm.cpp

```diff
@@ -28,7 +28,15 @@
 
 namespace fasm {
 
-FasmWriterVisitor::FasmWriterVisitor(std::ostream& f) : os_(f) {}
+FasmWriterVisitor::FasmWriterVisitor(vtr::string_internment *strings, std::ostream& f) : strings_(strings), os_(f),
+    fasm_lut(strings->intern_string(vtr::string_view("fasm_lut"))),
+    fasm_features(strings->intern_string(vtr::string_view("fasm_features"))),
+    fasm_params(strings->intern_string(vtr::string_view("fasm_params"))),
+    fasm_prefix(strings->intern_string(vtr::string_view("fasm_prefix"))),
+    fasm_placeholders(strings->intern_string(vtr::string_view("fasm_placeholders"))),
+    fasm_type(strings->intern_string(vtr::string_view("fasm_type"))),
+    fasm_mux(strings->intern_string(vtr::string_view("fasm_mux"))) {
+}
```

Comment:
Building static `intern_string`s is verbose, and used in many places. Can we use a macro or something to make it more concise?

Reply:
I could add a method to string internment for `const char *`, but I think that is a mistake because if you have a `string_view`, it is always preferable to use `string_view` over a raw `const char *`, and compute the strlen.

It is worth noting that I can likely update the `vtr::string_view(const char *)` to be `constexpr`, but I'm not worried about that cost right now.